src/test/compile-fail/private-in-public-warn.rs

```diff
@@ -287,5 +288,15 @@ mod aliases_params {
     type Result<T> = ::std::result::Result<T, Priv>;
 }
 
+mod foo {
+    struct Priv;
+    pub mod bar {
+        use super::Priv;
+        pub(super) fn f(_: Priv) {} // ok
+        pub(crate) fn g(_: Priv) {} //~ WARN private
```

Comment:
Heh, it's unfortunate that completely new errors are reported as warnings just because they are absent in the Old Error Set.